the-collab-lab/tcl-23-smart-shopping-list
# img added to Home.js and List.js

## Description

Added two images found from [Shape.so](https://shape.so/). I did not spend time working on alignment since that is addressed in future PRs. This PR probably could have been combined with a previous PR but here we are 😄 


## Related Issue

closes #49 


## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |


## Updates


<img width="250" alt="shoppingListImg" src="https://user-images.githubusercontent.com/38255956/120079153-3f90e100-c078-11eb-9c96-c5f8ca681780.png">



<img width="250" alt="writingHandImg" src="https://user-images.githubusercontent.com/38255956/120079146-37d13c80-c078-11eb-93a3-3d015d6536f5.png">


## Testing Steps / QA Criteria

1. On main run `git pull`
2. Run `git checkout add-shape-images`
3. Run `npm install`
4. Run `npm start`
5. Remove any local storage tokens
6. Refresh to `localhost:3000` to see shoppingBasket img
7. Create new list
8. See writing list image
